review/fileToReview.js

```diff
@@ -54,20 +41,40 @@ async function fetchTransactions(fromDate, authorization, jws = null, id, page,
             } else {
               throw new Error("Empty list of transactions ! " + JSON.stringify(previousTransactions));
             }
-            let nextPagesTransactions = fetchTransactions(fromDate, authorization, (jws || null), id, page + 1, mouvements);
+            let nextPagesTransactions = fetchTransactions(
+              fromDate,
+              authorization,
+              (jws || null),
+              id,
+              page + 1,
+              mouvements,
+            );
             response.data.Mouvements = mouvements.concat(nextPagesTransactions);
           }
         }
       }
       return response.data.Mouvements;
-    } else throw new Error();
-
-    return [];
```

Comment:
This code is unreachable

Reply:
La dernière instruction du (supposé) if est un return, l'instruction "return [];" est effectivement inatteignable.